src/custom/theme/baseTheme.tsx

```diff
@@ -23,19 +24,30 @@ export function colors(darkMode: boolean): Colors {
   return {
     ...colorsUniswap(darkMode),
 
-    // ****** base ******
-    white: darkMode ? '#c5daef' : '#ffffff',
-    black: darkMode ? '#021E34' : '#000000',
+    // CoW Swap V2 colors ======================
+    white: darkMode ? '#CAE9FF' : '#ffffff',
+    black: '#07162D',
+    blueDark1: '#07162D',
+    blueDark2: '#052B65',
+    blueLight1: '#CAE9FF',
+    grey1: darkMode ? '#07162D' : '#ECF1F8',
```

Comment:
Is it a good practice to add "colors" as names in the vars? shouldn't be more semantic instead?

Reply:
Yes makes sense, though that makes some of those names really long.